manifests/base/manager/deployment.yaml

```
@@ -0,0 +1,77 @@
+#
+# Copyright 2024 The Kubeflow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kubeflow-trainer-controller-manager
+  namespace: kubeflow-system
+  labels:
+    app.kubernetes.io/name: kubeflow-trainer
+    app.kubernetes.io/instance: kubeflow-trainer
+    app.kubernetes.io/version: "2.0.0"
+    app.kubernetes.io/managed-by: Kustomize
+    app.kubernetes.io/part-of: kubeflow
+    app.kubernetes.io/component: manager
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: kubeflow-trainer
+      app.kubernetes.io/instance: kubeflow-trainer
+      app.kubernetes.io/part-of: kubeflow
+      app.kubernetes.io/component: manager
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: kubeflow-trainer
+        app.kubernetes.io/instance: kubeflow-trainer
+        app.kubernetes.io/part-of: kubeflow
+        app.kubernetes.io/component: manager
+    spec:
+      containers:
+        - name: manager
+          image: kubeflow/trainer-controller-manager:2.0.0
```

This tag doesn't exist, and we should not add tag to the base manifests.
```suggestion
          image: kubeflow/trainer-controller-manager
```